lib/rules/no-restricted-imports.js

```diff
@@ -35,6 +35,12 @@ const arrayOfStringsOrObjects = {
                     message: {
                         type: "string",
                         minLength: 1
+                    },
+                    objectNames: {
```

Comment:
I think we should update this option name to be more precise. The values that this refers to aren't always objects (e.g. they could be numbers), and I don't think `objectNames` provides an intuitive description of what the option refers to.

Apparently the technical term for these are "Import Names" ([spec](https://tc39.github.io/ecma262/#importentry-record)), so I think an option like `importNames` would be better than `objectNames`.

Reply:
Beautiful! I will make this update.  That is a much better name.

Reply:
As you will see in the third commit, I have made the following renaming:

- importName -> importSource
- objectNames -> importNames

EX:
`const importSource = node.source.value.trim();`
`const importNames = node.specifiers.map(specifier => {`

Let me know what you think of the changes in the third commit.  Next, I will continue working on the rest of the review feedback.